.github/workflows/mcad-release.yml

```diff
@@ -0,0 +1,43 @@
+# This workflow will create a GitHub release tag
+
+name: mcad Release
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Release tag'
+        required: true
+        default: 'v0.0.0-dev'
+        type: string
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: checkout code
+      uses: actions/checkout@v3
+
+    - name: Configure Git
+      run: |
+        git config user.name "GitHub Actions"
+        git config user.email "github-actions@github.com"
```

Comment:
Is this action needed?
I think it has a meaning just for git commits.

Reply:
Yes its not required. i have added this earlier as part of branch creation will remove this